docs/features.md

```diff
@@ -39,6 +39,8 @@ The current branch is underlined in a branch layout.
 
 Git Machete can display commits belonging to the branches.
 It displays only the commits that are unique to the selected branch (i.e. have never been a tip of a different branch).
+Commits are displayed above their branch name, starting from the oldest ones
+(the newest commit is displayed just above the branch name).
```

Comment:
```suggestion
Commits are displayed above their branch name, starting from the oldest ones, with the newest commit displayed just above the branch name.
```

Reply:
(suggestions generally to be applied on a PR to be merged to `develop`... if you could tend to that @LukaszKontowski

Reply:
Sure